src/apify_client/clients/resource_clients/schedule.py

```diff
@@ -0,0 +1,83 @@
+from typing import Any, Dict, List, Optional
+
+from ..._errors import ApifyApiError
+from ..._utils import _catch_not_found_or_throw, _pluck_data_as_list, _snake_case_to_camel_case
+from ..base.resource_client import ResourceClient
+
+
+class ScheduleClient(ResourceClient):
+    """Sub-client for manipulating schedules."""
+
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
+        """Initialize the ScheduleClient."""
+        super().__init__(*args, resource_path='schedules', **kwargs)
+
+    def get(self) -> Optional[Dict]:
+        """Return information about schedule.
+
+        https://docs.apify.com/api/v2#/reference/schedules/schedule-object/get-schedule
+
+        Returns:
+            The retrieved schedule
+        """
+        return self._get()
+
+    def update(
+        self,
+        *,
+        cron_expression: Optional[str] = None,
+        is_enabled: Optional[bool] = None,
+        is_exclusive: Optional[bool] = None,
+        name: Optional[str] = None,
+        actions: Optional[List[Dict]] = None,
+        description: Optional[str] = None,
+        timezone: Optional[str] = None,
+    ) -> Optional[Dict]:
```

Comment:
We should make a decision about the optionality of the response for all the endpoint clients, so that it is consistent.

Reply:
I agree. And actually, `self._update` returns `Dict`, so I can use `Dict` here